tests/parm/gocart/AERO_ExtData.rc

```diff
@@ -1,99 +1,81 @@
+TROPP            'Pa'    Y       N                  -            0.0      1.0      TROPP         /dev/null:10000.
+
+#====== Dust Imports =================================================
 # FENGSHA input files. Note: regridding should be N or E
-DU_CLAY                 '1'  Y E - none none clayfrac     ExtData/dust/FENGSHA_DUST_INPUTS_v0.6.nc
-DU_SAND                 '1'  Y E - none none sandfrac     ExtData/dust/FENGSHA_DUST_INPUTS_v0.6.nc
-DU_SILT                 '1'  Y E - none none siltfrac     ExtData/dust/FENGSHA_DUST_INPUTS_v0.6.nc
-DU_SSM                  '1'  Y E - none none ssm          ExtData/dust/FENGSHA_DUST_INPUTS_v0.6.nc
-DU_RDRAG                '1'  Y E - none none drag_part    ExtData/dust/FENGSHA_DUST_INPUTS_v0.6.nc
-DU_UTHRES               '1'  Y E - none none uthres       ExtData/dust/FENGSHA_DUST_INPUTS_v0.6.nc
-
-# sulfate
-SU_regionMask        '1'                  Y        N               0              0.0      1.0     SU_regionMask     /dev/null:1.00
-pSO2_OCS             '1'                  Y        N               0              0.0      1.0     pSO2_OCS          /dev/null:0.00
-SU_AIRCRAFT          '1'                  Y        N               0              0.0      1.0     SU_AIRCRAFT       /dev/null:0.00
-SU_BIOMASS           '1'                  Y        N               0              0.0      1.0     SU_BIOMASS        /dev/null:0.00
-SU_ANTHROL1          '1'                  Y        N               0              0.0      1.0     SU_ANTHROL1       /dev/null:0.00
-SU_ANTHROL2          '1'                  Y        N               0              0.0      1.0     SU_ANTHROL2       /dev/null:0.00
-SU_SHIPSO2           '1'                  Y        N               0              0.0      1.0     SU_SHIPSO2        /dev/null:0.00
-SU_SHIPSO4           '1'                  Y        N               0              0.0      1.0     SU_SHIPSO4        /dev/null:0.00
-SU_DMSO              '1'                  Y        N               0              0.0      1.0     SU_DMSO           /dev/null:0.00
-SU_H2O2              '1'                  Y        N               0              0.0      1.0     SU_DMSO           /dev/null:0.00
-SU_NO3               '1'                  Y        N               0              0.0      1.0     SU_DMSO           /dev/null:0.00
-SU_OH                '1'                  Y        N               0              0.0      1.0     SU_DMSO           /dev/null:0.00
-SU_AVIATION_LTO      '1'                  Y        N               0              0.0      1.0     SU_AVIATION_LTO   /dev/null:0.00
-SU_AVIATION_CDS      '1'                  Y        N               0              0.0      1.0     SU_AVIATION_CDS   /dev/null:0.00
-SU_AVIATION_CRS      '1'                  Y        N               0              0.0      1.0     SU_AVIATION_CRS   /dev/null:0.00
-
-# CA
+DU_CLAY           '1'  Y E           -           none none clayfrac   ExtData/dust/FENGSHA_SOILGRIDS2019_GEFSv12_v1.2.nc
```

Comment:
@bbakernoaa Can this file be also used by other resolutions (C192/C394) runs too?